client/components/form.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+
+class UserForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      gender: '',
+      age: '',
+      height: '',
+      currentWeight: '',
+      goalWeight: '',
+      activityLevel: 'lightly active'
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    const { name, value } = event.target;
+    this.setState({
+      [name]: value
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const req = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(this.state)
+    };
+    fetch('/api/users', req)
+      .then(res => res.json())
+      .catch(error => console.error(error));
+  }
+
+  render() {
+    return (
+    <div className="form-container">
+      <form onSubmit={this.handleSubmit}>
+        <div className="row">
+          <div className="col-half">
+        <label>Gender</label>
+        <input
+        required
+        type="text"
+        onChange={this.handleChange}
+        name="gender"
+        placeholder="Female/Male">
+
+        </input>
+        </div>
+
+        <div className="col-half">
+        <label>Age</label>
+        <input
+        required
+        type="number"
+        onChange={this.handleChange}
+        min="18"
+        max="100"
+        name="age">
+        </input>
+        </div>
+        </div>
+
+        <div className="row">
+        <div className="col-half">
+        <label>Height</label>
+        <input
+        required
+        type="text"
+        onChange={this.handleChange}
+        name="height"
+        placeholder="Inches">
+        </input>
+        </div>
+
+        <div className="col-half">
+        <label>Current Weight</label>
+        <input
+        required
+        type="number"
+        onChange={this.handleChange}
+        min="100"
+        name="currentWeight"
+        placeholder="Pounds">
+        </input>
+        </div>
```

Comment:
This indentation is very hard to follow. Confirm if you linter is working, but child elements should always be indented within their parent.